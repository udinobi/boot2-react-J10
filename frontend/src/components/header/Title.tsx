import styled from "styled-components"

export default styled.span`
    color: #ffffff;
    @media (max-width: 379px) {
        font-size: 1.3rem;
    }
    @media (min-width: 380px) {
        font-size: 1.6rem;
    }
    margin-left: 1%;
`
