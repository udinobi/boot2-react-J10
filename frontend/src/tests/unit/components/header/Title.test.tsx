import React from "react"
import * as Renderer from 'react-test-renderer'

import Title from "../../../../components/header/Title"

describe("--> Snapshot for Components > header > 'Title'", () => {
    it("matches", () => {
        const component = Renderer.create(<Title />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
