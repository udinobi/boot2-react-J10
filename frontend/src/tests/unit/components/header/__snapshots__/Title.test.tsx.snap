// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`--> Snapshot for Components > header > 'Title' matches 1`] = `
<span
  className="sc-bdVaJa dvkyIH"
/>
`;
